helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -137,6 +137,14 @@ def create(self, vals_list):
                 vals["team_id"] = self._prepare_team_id(vals)
             if vals.get("user_id") and not vals.get("assigned_date"):
                 vals["assigned_date"] = fields.Datetime.now()
+            # Automatically set e-mail channel when created from the
+            # fetchmail cron task
+            if self.env.context.get("fetchmail_cron_running") and not vals.get(
+                "channel_id"
+            ):
+                vals["channel_id"] = self.env.ref(
+                    "helpdesk_mgmt.helpdesk_ticket_channel_email"
+                ).id
```

Comment:
Protect this call with second argument `False` for avoiding an error if the channel is removed.